### PR TITLE
Add Eigen3

### PR DESCRIPTION
**Pull request by dsendkowski**

### CODE REVIEW

File: components/scientific/eigen3/Makefile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#
+
+#
+# Copyright 2017 Dariusz Sendkowski
+#
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=           eigen
+COMPONENT_VERSION=        3.3.3
+COMPONENT_FMRI=           library/eigen
```
Comment (alarcher) [inline components/scientific/eigen3/Makefile]: Should the package name be 'library/eigen3'? It seems that other systems have one package per major version.

File: components/scientific/eigen3/Makefile [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+include $(WS_MAKE_RULES)/cmake.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+
+
+build:		$(BUILD_32)
+
+install:	$(INSTALL_32)
+
+test:		$(NO_TEST)
```
Comment (pyhalov) [inline components/scientific/eigen3/Makefile:39]: Any specific build  requirements?

Comment (dsendkowski) [reply components/scientific/eigen3/Makefile:39]: Actually, there is no build. This is a bunch of header files to be installed.

File: components/scientific/eigen3/Makefile (hunk 1 above)
Comment (dsendkowski) [reply components/scientific/eigen3/Makefile]: Yeah, I've changed it.

**Pull request merged**